sw/airborne/modules/computer_vision/video_thread.c

```diff
@@ -80,6 +80,9 @@ static void *video_thread_function(void *data)
 {
   struct video_config_t *vid = (struct video_config_t *)data;
 
+  char print_tag[80];
+  sprintf(print_tag, "video_thread-%s", vid->dev_name);
```

Comment:
You still have the option to only copy/save the image in the callback from the video_thread and then run the processing of it in another thread or schedule it as you want...

There is no requirement to run the processing in the callback (and hence the video_thread thread) unless you want to pass a modified version on to the next in the pipeline.